src/react-modules/pages/CharacterPage.tsx

```diff
@@ -0,0 +1,28 @@
+import { ShowcaseJSON } from '../../types';
+import {CharacterScreen} from '../character-screen/CharacterScreen'
+
+export const CharacterPage = ({
```

Comment:
I have been seeing these same arguments over and over in the PRs it seems. Probably makes sense to create an interface out of it, so we are sure that all these functions are really using params in a uniform way